pkg/xds/envoy/secrets/v3/ca_secret.go

```diff
@@ -10,9 +10,14 @@ import (
 	"github.com/kumahq/kuma/pkg/xds/envoy/tls"
 )
 
-func CreateCaSecret(secret *core_xds.CaSecret) *envoy_auth.Secret {
+func CreateCaSecret(secret *core_xds.CaSecret, mesh string) *envoy_auth.Secret {
+	name := tls.MeshCaResource
+	if mesh != "" {
+		name += "_" + mesh
```

Comment:
We have a file `pkg/xds/envoy/names/resource_names.go` that contains a lot of functions for name constructions. I believe it's better to move this logic to the file

Reply:
done, I reused `names.GetSecretName`, which changed the names from `mesh_ca_[mesh]` to `mesh_ca:secret:[mesh]` and from `identity_cert_[mesh]` to `identity_cert:secret:[mesh]` which I think is more consistent with other code we use